tgstation.dme

```diff
@@ -6,6 +6,51 @@
 
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "html"

```

Comment:
You need to go to Build > Preferences and untick 'Automatically set FILE_DIR for sub-directories', then recompile and update this PR.